include/nitro/broken_options/option/toggle.hpp

```diff
@@ -66,11 +66,44 @@ namespace broken_options
             return given_;
         }
 
+        int default_value(const bool& def)
```

Comment:
Usually, you don't need to pass primitive types with const references. You can just pass them by value.

For a reasonable default, have a look at Slide 24 of [this](https://github.com/CppCon/CppCon2014/blob/master/Presentations/Back%20to%20the%20Basics!%20Essentials%20of%20Modern%20C%2B%2B%20Style/Back%20to%20the%20Basics!%20Essentials%20of%20Modern%20C%2B%2B%20Style%20-%20Herb%20Sutter%20-%20CppCon%202014.pdf)